### ISSUE TITLE
warning about missing @vue/compiler-sfc@3.x when installing on a svelte-kit project

### ISSUE THREAD

**Issue by opensas** [trivago/prettier-plugin-sort-imports#201]
**Your Environment**
 * **Prettier version**: 2.8.1
 * **node version** [v16.15.0]:
 * **package manager**: [pnpm@7.17.0]
 * **IDE**: [VScode]

```
devDependencies:
  '@playwright/test': 1.29.1
  '@sveltejs/adapter-auto': 1.0.0_@sveltejs+kit@1.0.1
  '@sveltejs/kit': 1.0.1_svelte@3.55.0+vite@4.0.4
  '@trivago/prettier-plugin-sort-imports': 4.0.0_prettier@2.8.1
  '@typescript-eslint/eslint-plugin': 5.48.0_k73wpmdolxikpyqun3p36akaaq
  '@typescript-eslint/parser': 5.48.0_iukboom6ndih5an6iafl45j2fe
  eslint: 8.31.0
  eslint-config-prettier: 8.6.0_eslint@8.31.0
  eslint-plugin-svelte3: 4.0.0_6go7qzsvzihesfwmyvq3tnl5qq
  prettier: 2.8.1
  prettier-plugin-svelte: 2.9.0_ajxj753sv7dbwexjherrch25ta
  svelte: 3.55.0
  svelte-check: 2.10.3_svelte@3.55.0
  tslib: 2.4.1
  typescript: 4.9.4
  vite: 4.0.4
  vitest: 0.25.8
```
**Describe the bug**

After installing the plugin the following warning is displayed:

```
devDependencies:
+ @trivago/prettier-plugin-sort-imports 4.0.0

 WARN  Issues with peer dependencies found
.
└─┬ @trivago/prettier-plugin-sort-imports 4.0.0
  └── ✕ missing peer @vue/compiler-sfc@3.x
Peer dependencies that should be installed:
  @vue/compiler-sfc@3.x 
  ````

**To Reproduce**

1. create new svelte-kit project

```
$ pnpm create svelte@latest tmp-app

create-svelte version 2.0.1

Welcome to SvelteKit!

✔ Which Svelte app template? › Skeleton project
✔ Add type checking with TypeScript? › Yes, using TypeScript syntax
✔ Add ESLint for code linting? … No / Yes
✔ Add Prettier for code formatting? … No / Yes
✔ Add Playwright for browser testing? … No / Yes
✔ Add Vitest for unit testing? … No / Yes

Your project is ready!
✔ Typescript
  Inside Svelte components, use <script lang="ts">
✔ ESLint
  https://github.com/sveltejs/eslint-plugin-svelte3
✔ Prettier
  https://prettier.io/docs/en/options.html
  https://github.com/sveltejs/prettier-plugin-svelte#options
✔ Playwright
  https://playwright.dev
✔ Vitest
  https://vitest.dev
  ```
  
2. install dependencies
  
```
$ cd tmp-app/
$ pnpm install
```

3. install plugin

```
$ pnpm install --save-dev @trivago/prettier-plugin-sort-imports
 WARN  deprecated sourcemap-codec@1.4.8: Please use @jridgewell/sourcemap-codec instead
Packages: +56
++++++++++++++++++++++++++++++++++++++++++++++++++++++++
Progress: resolved 302, reused 280, downloaded 0, added 56, done

devDependencies:
+ @trivago/prettier-plugin-sort-imports 4.0.0

 WARN  Issues with peer dependencies found
.
└─┬ @trivago/prettier-plugin-sort-imports 4.0.0
  └── ✕ missing peer @vue/compiler-sfc@3.x
Peer dependencies that should be installed:
  @vue/compiler-sfc@3.x  

Done in 4.5s
````

**Expected behavior**

no warning message

```
$ cat .prettierrc 
{
	"useTabs": true,
	"singleQuote": true,
	"trailingComma": "none",
	"printWidth": 100,
	"plugins": ["prettier-plugin-svelte"],
	"pluginSearchDirs": ["."],
	"overrides": [{ "files": "*.svelte", "options": { "parser": "svelte" } }]
}
```

```
$ cat .prettierignore 
.DS_Store
node_modules
/build
/.svelte-kit
/package
.env
.env.*
!.env.example

# Ignore files for PNPM, NPM and YARN
pnpm-lock.yaml
package-lock.json
yarn.lock
```

**Comment by cdbattags** [comment]
I get the same for a plain ol' Next.js/React app as well. Is there any way to conditionally make peer deps?

**Comment by aligorgin** [comment]
yup ! i get the same thing in the next.js app. 

` WARN  Issues with peer dependencies found
.
└─┬ @trivago/prettier-plugin-sort-imports 4.0.0
  └── ✕ missing peer @vue/compiler-sfc@3.x
Peer dependencies that should be installed:
  @vue/compiler-sfc@3.x`

**Comment by aligorgin** [comment]
yep ! I get the same thing in the Next,js app .

` WARN  Issues with peer dependencies found`
`.`
`└─┬ @trivago/prettier-plugin-sort-imports 4.0.0`
` └── ✕ missing peer @vue/compiler-sfc@3.x`
`Peer dependencies that should be installed:`
`  @vue/compiler-sfc@3.x`

**Comment by aligorgin** [comment]
yep! I got the same thing on Next.js app.

```sh
 WARN  Issues with peer dependencies found
.
└─┬ @trivago/prettier-plugin-sort-imports 4.0.0
  └── ✕ missing peer @vue/compiler-sfc@3.x
Peer dependencies that should be installed:
  @vue/compiler-sfc@3.x
```

**Comment by robertknight** [comment]
The  @vue/compiler-sfc@3.x dependency was marked as optional in https://github.com/trivago/prettier-plugin-sort-imports/commit/a5343b54063cda43132cc28fe776ae40c7e142e4. However that change has not yet been included in a published version.

Could we have a new release of this package with that change please? Thank-you!

**Comment by byara** [comment]
The issue should be resolved in v4.1.0
